### PR TITLE
open-adventure 1.9 (new formula)

### PR DESCRIPTION
**Pull request by danavery**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/open-adventure.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  resource "PyYAML" do
+    url "https://files.pythonhosted.org/packages/a0/a4/d63f2d7597e1a4b55aa3b4d6c5b029991d3b824b5bd331af8d4ab1ed687d/PyYAML-5.4.1.tar.gz"
+    sha256 "607774cbba28732bfa802b54baa7484215f530991055bb562efbed5b2f20a45e"
+  end
+
+  def install
+    venv = virtualenv_create(libexec, "python3.9")
+    venv.pip_install resources
+    system("#{libexec}/bin/python", "./make_dungeon.py")
```
Comment (SMillerDev) [inline Formula/open-adventure.rb:23]:
```suggestion
    system libexec/"bin/python", "./make_dungeon.py"
```

File: Formula/open-adventure.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    system "make"
+    bin.install "./advent"
```
Comment (SMillerDev) [inline Formula/open-adventure.rb:25]:
```suggestion
    bin.install "advent"
```

File: Formula/open-adventure.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    system "make", "advent.6"
+    man6.install "./advent.6"
```
Comment (SMillerDev) [inline Formula/open-adventure.rb:27]:
```suggestion
    man6.install "advent.6"
```

Comment (danavery) [reply Formula/open-adventure.rb]: 👍

**Comment by jonchang** [comment]
Let's add a `head` for https://gitlab.com/esr/open-adventure so livecheck can pick up potential updates. :+1: nice work and thanks for contributing to Homebrew!

**Review by chenrui333 — APPROVED** [review]

**Comment by jonchang** [comment]
Hooray for classic games! Thanks for contributing to Homebrew, @danavery!

**Pull request closed without merging**